src/App.vue

```diff
@@ -20,6 +20,9 @@
     },
   })
   export default class App extends Vue {
+    async fetchPageData() {
+      this.$vxm.user.authenticate();
+    }
```

Comment:
This doesn't call it for me - why would `fetchPageData` would be called on `App`?